### ISSUE TITLE
Memory shows negative numbers if not 0% mem usage

### ISSUE THREAD

**Issue by djaydev** [amir20/dozzle#2587]
**Describe the bug**
Memory shows negative numbers if not 0%

**To Reproduce**
open dozzle in chrome web browser

**Screenshots**
![Screenshot 2023-12-11 105926](https://github.com/amir20/dozzle/assets/33335484/7589cd83-610e-4667-bf94-47f700cbd0b5)

**Desktop (please complete the following information):**
- OS: alpine linux
- Docker version
```
:~$ docker version
Client:
 Version:           24.0.7
 API version:       1.43
 Go version:        go1.21.3
 Git commit:        afdd53b4e341be38d2056a42113b938559bb1d94
 Built:             Fri Oct 27 15:13:04 2023
 OS/Arch:           linux/amd64
 Context:           default

Server:
 Engine:
  Version:          24.0.7
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.21.3
  Git commit:       311b9ff0aa93aa55880e1e5f8871c4fb69583426
  Built:            Fri Oct 27 15:13:04 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.7.10
  GitCommit:        4e1fe7492b9df85914c389d1f15a3ceedbb280ac
 runc:
  Version:          1.1.10
  GitCommit:        18a0cb0f32bcac2ecc9a10f327d282759c144dab
 docker-init:
  Version:          0.19.0
  GitCommit:
```
- Browser & version - chrome Version 120.0.6099.71 (Official Build) (64-bit)
- Version v5.7.0

**If applicable include logs with `--level debug` and browser logs**

**Comment by djaydev** [comment]
Hi amir20,
No I'm using an Intel NUC 12 Extreme, 12th GEN x86 CPU. Alpine linux is installed bare metal, then just dozzle:latest image running in docker.

**Comment by djaydev** [comment]
sorry again just going to close this. After some testing in other servers its something outside of dozzle

**Comment by joshdick** [comment]
I'm having the same issue as @djaydev with Docker version 24.0.7 installed on Alpine Linux x86_64 bare metal (also presumably the same as @djaydev.)

**Comment by djaydev** [comment]
@joshdick the issue is with runc / cgroup in Alpine. The lead guy over there is aware of it, they got a few issues open on it.

**Comment by joshdick** [comment]
Just to follow up, I can confirm that this issue is now [fixed in Alpine 3.19.0](https://git.alpinelinux.org/aports/commit/?id=06e895bc94fd56101fd27d9a2571b0ac0571300c) thanks to backporting an upstream fix in openrc.
